x-pack/test/functional/services/ml/trained_models_table.ts

```diff
@@ -264,6 +527,25 @@ export function TrainedModelsTableProvider(
       );
     }
 
+    public async deployModelsContinue(expectedStep?: string) {
+      await testSubjects.exists('mlTrainedModelsInferencePipelineContinueButton');
+      const continueButton = await testSubjects.find(
+        'mlTrainedModelsInferencePipelineContinueButton'
+      );
+      await continueButton.isEnabled();
```

Comment:
this line doesn't affect test execution. FYI it doesn't wait for button to be enabled. 
```suggestion
```